EIPS/Skinny_CREATE2.md

```diff
@@ -0,0 +1,16 @@
+```
```

Comment:
Please update this to the new format: frontmatter should start and end with `---` on its own line, and keys are lower-case.